optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/migrate313to86/PreventOptimize8UpgradeOnOptimize7IT.java.out

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+//package io.camunda.optimize.upgrade.migrate313to86;
+//
+//import static org.assertj.core.api.Assertions.assertThatCode;
+//import static org.assertj.core.api.Assertions.assertThatThrownBy;
+//
+//import io.camunda.optimize.upgrade.exception.UpgradeRuntimeException;
+//import org.junit.jupiter.api.Test;
+//
+//public class PreventOptimize8UpgradeOnOptimize7IT extends AbstractUpgrade86IT {
```

Comment:
I don't know why this is commented out, should it not be included here?

Reply:
Upgrade IT are not yet in the monorepo. I am still working on migrating those in the other PR which is higher effort than anticipated since some OS changes on main broke the PR after rebasing. I had scheduled a call with giualiano to pair on this before I got sick